docker-entrypoint.sh

```diff
@@ -410,6 +418,8 @@ if [ "$haveSslConfig" ] && [ -f "$combinedSsl" ]; then
 	sslErlArgs="-pa $ERL_SSL_PATH -proto_dist inet_tls -ssl_dist_opt server_certfile $combinedSsl -ssl_dist_opt server_secure_renegotiate true client_secure_renegotiate true"
 	export RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS="${RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS:-} $sslErlArgs"
 	export RABBITMQ_CTL_ERL_ARGS="${RABBITMQ_CTL_ERL_ARGS:-} $sslErlArgs"
+
+	# TODO show a warning that we set ERL_SSL_PATH, RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS, and RABBITMQ_CTL_ERL_ARGS (with their set values)
```

Comment:
`RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS` & `RABBITMQ_CTL_ERL_ARGS` environment variables should be allowed. The docker-entrypoint shouldn't need to do anything about them though.

This contains all environment variables that get used before the Erlang VM starts, most of them used to configure the Erlang VM: https://github.com/rabbitmq/rabbitmq-server/blob/v3.8.5/scripts/rabbitmq-env

Here is a comprehensive list of all environment variables that RabbitMQ reads during the boot phase, after the Erlang VM starts: https://github.com/rabbitmq/rabbitmq-common/blob/v3.8.5/src/rabbit_env.erl#L39-L74

Reply:
I guess this and the "combined" file creation are really two parts of the same whole -- we only set these if we created a combined file, and these are set so that said combined file gets used everywhere, so I think we're fully in agreement there. :+1: